implementations/rust/ockam/ockam_command/src/tcp/outlet/show.rs

```diff
@@ -39,6 +44,13 @@ impl ShowCommand {
     }
 }
 
+#[derive(Debug, Serialize)]
+struct OutletInformation {
+    alias: String,
+    addr: MultiAddr,
+    socket_addr: SocketAddr,
+}
```

Comment:
Now that you have a specific type to handle the output, you could go a step further and implement the `Output` trait. You have an example on how to do that [here](https://github.com/build-trust/ockam/blob/60627b3b627b03ef068fa7bb44428e9f99635679/implementations/rust/ockam/ockam_command/src/output/output.rs#L142).

Once yo have that, you could do `let plain = info.output()?;` in the command.

Reply:
Perfect! Updated.